### DISCUSSION TITLE
Can someone confirm this image url can be downloaded in Python or curl?

### DISCUSSION THREAD

**Discussion by davidbernat** [tweepy/tweepy#2031] [Questions] [answered]
Often the embedded image will be hosted internally to Twitter, with a `expanded_url` such as:

`https://twitter.com/POTUS/status/1610721959330840578/photo/1`

Can someone confirm for me this image can be downloaded using Python `requests` or command line `curl` and post their code in the reply?

Thank you 🙏

**Comment by Harmon758** [comment, marked as answer]
That's simply the display page for the media.
You'll want to use the media URL instead.

**Reply by davidbernat** [reply]
Gosh. My mistake. Thank you. 🙏 
The correct location is in `includes['media'][{index}]['url']`.
